src/modm/platform/gpio/sam/unused.hpp

```diff
@@ -60,15 +61,6 @@ class GpioUnused : public Gpio, public ::modm::GpioIO
 	// end documentation inherited
 	static void configure(InputType) {}
 	static void setInput(InputType) {}
-	// External Interrupts
-	static void enableExternalInterrupt() {}
-	static void disableExternalInterrupt() {}
-	static void enableExternalInterruptVector(const uint32_t) {}
-	static void disableExternalInterruptVector() {}
-	static void setInputTrigger(const InputTrigger) {}
-	static bool getExternalInterruptFlag() { return false; }
-	/// Reset the interrupt flag in the interrupt routine.
-	static void acknowledgeExternalInterruptFlag() {}
```

Comment:
Why are all the functions removed from `GpioUnused`?

Reply:
The SAM port has its own separate peripheral for external interrupts, thus this API is not part of the (platform-specific) GPIO API.